src/lib/apiErrorHandler.js

```diff
@@ -3,48 +3,71 @@ import Sequelize from 'sequelize';
 import createRequestError from '../services/requestErrors';
 import { auditLogger as logger } from '../logger';
 
-/**
- * Handles sequelize errors
- *
- * @param {*} req - request
- * @param {*} res - response
- * @param {*} error - error
- * @param {*} logContext - useful data for logging
- */
-async function handleSequelizeError(req, res, error, logContext) {
+async function logRequestError(req, operation, error, logContext) {
+  if (operation !== 'SequelizeError'
+  && process.env.SUPPRESS_ERROR_LOGGING
+  && process.env.SUPPRESS_ERROR_LOGGING.toLowerCase() === 'true') {
+    return 0;
+  }
```

Comment:
Thank you for adding the environment variable. 👍  To make it effective (per env) we still need to add it to the configuration files.

Reply:
which config file are you referring to

Reply:
I think i fixed this